libs/nuxt/src/builders/server/builder.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  BuilderContext,
+  BuilderOutput,
+  createBuilder,
+  targetFromTargetString,
+} from '@angular-devkit/architect';
+import { join, JsonObject, normalize } from '@angular-devkit/core';
+import { build, loadNuxt } from 'nuxt';
+import { concat, from, Observable } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { ServerBuilderSchema } from './schema';
+import { BrowserBuilderSchema } from '../browser/schema';
+import { getProjectRoot } from '../../utils';
+
+const serverBuilderOverriddenKeys = [];
+
+export function runBuilder(
+  options: ServerBuilderSchema,
+  context: BuilderContext
+): Observable<BuilderOutput> {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  async function setup(): Promise<any> {
+    const browserTarget = targetFromTargetString(options.browserTarget);
+    const rawBrowserOptions = await context.getTargetOptions(browserTarget);
+    const overrides = Object.keys(options)
+      .filter(
+        (key) =>
+          options[key] !== undefined &&
+          serverBuilderOverriddenKeys.includes(key)
+      )
+      .reduce((previous, key) => ({ ...previous, [key]: options[key] }), {});
+    const browserName = await context.getBuilderNameForTarget(browserTarget);
+    const browserOptions = await context.validateOptions<
+      JsonObject & BrowserBuilderSchema
+    >({ ...rawBrowserOptions, ...overrides }, browserName);
+
+    const projectRoot = await getProjectRoot(context);
+
+    const nuxt = await loadNuxt({
+      for: options.dev ? 'dev' : 'start',
+      rootDir: projectRoot,
```

Comment:
Any file paths passed to `nuxt` should be transformed with `getSystemPath`. Tested on Windows and the pages do not load without it.